tendrl/commons/objects/definition/master.yaml

```diff
@@ -384,60 +351,6 @@ namespace.tendrl:
       enabled: true
       value: nodes/$NodeContext.node_id/Os
       help: "OS"
-    Package:
```

Comment:
"Package" object not required? https://github.com/Tendrl/commons/blob/develop/tendrl/commons/objects/package/__init__.py